backend/schema/jsonschema.go

```diff
@@ -161,6 +163,13 @@ func nodeToJSSchema(node Node, refs map[RefKey]*Ref) *jsonschema.Schema {
 	case *TypeParameter:
 		return &jsonschema.Schema{}
 
+	case *SumType:
+		variants := make([]jsonschema.SchemaOrBool, len(node.Variants))
+		for i, v := range node.Variants {
+			variants[i] = jsonschema.SchemaOrBool{TypeObject: nodeToJSSchema(v, refs)}
+		}
+		return &jsonschema.Schema{OneOf: variants}
```

Comment:
Nice!

Though, this should be using `{"kind": xyz, "value": ...}`, or is that in a followup?

Reply:
fyi - from feedback during the design sync we actually lifted the original struct out of `value`, so it'll just be an additional field named `@_kind` (prefixed with `@_` to avoid conflicts) at the top level now

Reply:
Mmm, `@_kind` can still conflict though if the sum type variant is a map:

```
```

Reply:
ah good point, i misunderstood your prior comment from the design sync. we can go with the kind/value approach then